app/assets/javascripts/taxonomy_edit.js

```diff
@@ -1,23 +1,34 @@
 $(function() {
-  $(':checkbox').each(function(i, item) {
-    ignore_checked(item);
+  $("input.ignore_types:not(.dual_list)").each(function(i, item) {
+    ignore_checked(item, false);
   });
 });
 
-function ignore_checked(item) {
-  var current_select = $(item)
-    .closest('.tab-pane')
-    .find('select');
+function ignore_checked(item, dual_list) {
+  if(dual_list){
+    var checkBox = $(item);
+    var dualList = checkBox.closest('.tab-pane').find('.dual-list');
 
-  if ($(item).is(':checked')) {
-    current_select.attr('disabled', 'disabled');
-  } else {
-    current_select.removeAttr('disabled');
-  }
-  if (!$(current_select).hasClass('parameter_type_selection')) {
-    $(current_select).multiSelect('refresh');
+    if (checkBox.is(':checked')) {
+      dualList.addClass('disabled');
```

Comment:
This is not correct aproach, we should not touch the content rendered by react component from jquery code, but we'd need to add more code to aproach this, so I'm ok with this, if you add an comment that it should be taken care of in the next PRs :)

Reply:
Changing React component with jquery is totally an anti-pattern.
React uses virtual-dom while jquery manipulates the real dom.
You can use [this](https://github.com/theforeman/foreman/pull/7019) for invoking an action from legacy js and trigger a rerender in the react side